server/athenian/api/models/state/models.py

```diff
@@ -167,3 +167,28 @@ class ReleaseSetting(create_time_mixin(updated_at=True), Base):
     branches = Column(String(1024))
     tags = Column(String(1024))
     match = Column(SmallInteger())
+
+
+class Feature(create_time_mixin(updated_at=True), Base):
```

Comment:
Do you think it would need to have also a `params` column? Or you think that it would be better to make a feature flat so that different params would be equivalent to different features? E.g.: a feature flag for limiting the number of teams. It could either a feature named `limit_teams_to_5` or `limt_teams` with a param column `{value: 5}`. If we want to just change the value we just need to update the db, in the other case, I'd expect the code somewhere to have something like:
```
if feature_enabled("limit_teams_to_5") and len(teams) >= 5:
    raise
```
so it would require changing the code.

Reply:
Let's a new column `parameters` of type JSON. It should be per-account rather than global though.